src/components/Form.svelte

```diff
@@ -43,7 +45,9 @@
     handle: /^[a-zA-Z0-9-_]{1,39}$/,
     address: /^0x[a-zA-Z0-9]{40}$/,
     id: /^[a-z0-9]+$/,
-    identity: /^rad:git:[a-z0-9]{37}$/,
+    identity: window.HEARTWOOD
+      ? /^rad:[a-z0-9]{37}$/
```

Comment:
Don't think it's 37 exactly, since we changed bases?

Reply:
Good catch: 8e3a0217e09eeff5abd3867c35c0b9189cfd535b.